src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -238,6 +242,13 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
           onChange={togglePersonalisedSchedule}
           label="Bookmarked events"
         />
+
+        <ButtonNG
+          className="NavBarWrapper__close-button"
+          iconName={faTimes}
+          onClick={() => closeEventSchedule()}
```

Comment:
```suggestion
          onClick={closeEventSchedule}
```

Reply:
fixed in 16d383be67e204b24617a265e9625a7cdcae0106